packages/flutter/lib/src/rendering/object.dart

```diff
@@ -859,6 +859,20 @@ class _LocalSemanticsHandle implements SemanticsHandle {
 /// are visible on screen. You can create other pipeline owners to manage
 /// off-screen objects, which can flush their pipelines independently of the
 /// on-screen render objects.
+///
+/// [PipelineOwner]s can be organized in a tree to manage multiple render trees,
```

Comment:
```suggestion
/// [PipelineOwner]s are organized in a tree to manage multiple render trees,
```

Aren't they always arranged this way (even a single main view can have subviews).

Reply:
I wanted to express her that the PipelineOwner class can be used stand-alone. It fully functions if it is not part of a tree, for example for the use case mentioned above where you create another pipeline owner to manage off-screen objects. You are not required to turn them into a tree or attach them to any (existing) tree. Of course, one could argue, that just having a single node (the root) is also a form of tree...

Reply:
Oh, OK, ignore my comment then. 

> Of course, one could argue, that just having a single node (the root) is also a form of tree...

We don't need to be as pedantic as all that. :-)